flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/batch/table/stringexpr/CalcStringExpressionTest.scala

```diff
@@ -362,4 +362,42 @@ class CalcStringExpressionTest extends TableTestBase {
 
     verifyTableEquals(t1, t2)
   }
+
+  @Test
+  def testAddColumns(): Unit = {
+    val util = batchTestUtil()
+    val t = util.addTable[(Int, Long, String)]("Table3",'a, 'b, 'c)
+
+    var t1 = t.addColumns(true, concat('c, "Sunny") as 'kid).addColumns('b + 1)
```

Comment:
Can we have the same tests for both `string` and `expression` approach? Best if we could unify them. Maybe sth similar to: `org.apache.flink.table.expressions.utils.ExpressionTestBase#testAllApis`?

Reply:
I think `ExpressionTestBase#testAllApis` is better for a function test, and operator tests, such as `select`, `filter` etc. using `CalcStringExpressionTest` is fine, from my side. What do you think?